packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -418,9 +424,40 @@ export class GeneratedAnswerCommon {
     );
   }
 
+  private renderCustomNoAnswerMessage() {
+    const {getBindings} = this.props;
+    const {i18n} = getBindings();
+
+    return (
+      <div part="generated-content">
+        <div class="flex items-center">
+          <Heading
+            level={0}
+            part="header-label"
+            class="text-bg-primary inline-block rounded-md px-2.5 py-2 font-medium"
+          >
+            {i18n.t('generated-answer-title')}
+          </Heading>
+        </div>
+        <div part="generated-container" class="mt-6">
+          <slot name="no-answer-message"></slot>
+        </div>
+      </div>
+    );
+  }
+
   public render() {
+    const {getGeneratedAnswerState} = this.props;
+    const {cannotAnswer} = getGeneratedAnswerState() ?? {};
+
     if (this.shouldBeHidden) {
```

Comment:
This might seems out of scope. But I cant help wondering if we should review the name of this condition, that now looks quite misleading.

Reply:
indeed! nice catch